react/Reviews.tsx

```diff
@@ -478,23 +485,23 @@ const Reviews: FunctionComponent<InjectedIntlProps & Props> = props => {
       >
         <FormattedMessage id="store/reviews.list.title" />
       </h3>
-      <div className="review__rating">
+      <div className={`${handles.reviewsRating} review__rating`}>
         {!state.hasTotal || !state.hasAverage ? (
           <FormattedMessage id="store/reviews.list.summary.loading" />
         ) : !state.total ? null : (
           <Fragment>
-            <div className="t-heading-4">
+            <div className={`${handles.starsContainer} t-heading-4`}>
               <Stars rating={state.average} />
             </div>
-            <span className="review__rating--average dib v-mid">
+            <span className={`${handles.reviewsRatingAverage} review__rating--average dib v-mid`}>
```

Comment:
Lint issue here (reported by the GitHub Action)

To setup the linter you should run:
-  `yarn` on the root directory
-  `yarn` on the `react` directory

Then run `yarn lint --fix` on the root directory to fix the issues.